server/events/models/models.go

```diff
@@ -71,7 +74,20 @@ func NewRepo(vcsHostType VCSHostType, repoFullName string, cloneURL string, vcsU
 		return Repo{}, errors.New("cloneURL can't be empty")
 	}
 
-	if !strings.HasSuffix(cloneURL, ".git") {
+	if vcsHostType == AzureDevops {
```

Comment:
This was failing for me because my repo had a `-` in it. The regex could be fixed but is this necessary? Can a malformed repo name get in here?